tests/integration/components/switch-test.js

```diff
@@ -164,9 +164,7 @@ module('Integration | Component | <Switch>', function (hooks) {
         assertSwitchState({ state: SwitchState.Off });
       });
     });
-  });
 
-  module('Keyboard interactions', function () {
```

Comment:
did you intend to remove this? AFAIK, this is mirroring the structure of the Vue tests

Reply:
I did! where was a qunit lint warning about a module _with the same name_ immediately preceding, so the module separation doesn't make sense, ya know?

Reply:
right here: https://github.com/GavinJoyce/ember-headlessui/pull/89/files/573fa499920dd3cf91c1c1c500181eced5aa3909..d3d7b203bebde0b7f61e7b8da88056eea3a43ae3#diff-f92b87d45171cafc52ed0f458af0ac7d6779551c40dd95ea796d902a8b913b5dR140

Reply:
👍 aha, nice warning from eslint